pennylane/ops/op_math/exp.py

```diff
@@ -193,6 +194,8 @@ def decomposition(self):
         Returns:
             list[PauliRot]: decomposition of the operator
         """
+        from pennylane.pauli import pauli_word_to_string  # pylint: disable=import-outside-toplevel
```

Comment:
Same here

Reply:
I tried it with that syntax first and it still throws the circular import error